examples/simple-genai-server/README.md

```diff
@@ -0,0 +1,75 @@
+# Simple Game Server for GenAI
+
+A simple game server that serves as an example of how to integrate GenAI endpoints into your
+dedicated game server. You can either interact with one GenAI endpoint via TCP, or set up two
+endpoints to "chat" to each other.
+
+## Setting up the GenAI Inference Server
+
+You will need a separate GenAI inference server. This game server uses the
+[Google for Games GenAI](https://github.com/googleforgames/GenAI-quickstart) as its inference server.
+This particular inference server has the request structure for the /chat endpoint:
+
+```
+type GenAIRequest struct {
+	Context         string  `json:"context,omitempty"`
+	Prompt          string  `json:"prompt"`
+}
+```
+
+If you need a different request structure, you will need to clone or fork this repository and
+modify this in `main.go`. Update `REPOSITORY` in the `agones/examples/simple-genai-server/Makefile`
+to your own container registry and run `make build && make push` from within the
+`agones/examples/simple-genai-server` path. Then modify the `gameserver.yaml` to pull the image from
+your container registry. If you are making a series of changes you may also want to add
+`imagePullPolicy: Always` to the container image in `gameserver.yaml`.
+
+## Setting up the Game Server
+
+This example uses two separate clusters for the GenAI server and the game server. For the game
```

Comment:
I'm wondering why we don't suggest running 1 autopilot cluster with everything in it. After setting up the GenAI cluster, you just install agones into it and launch the game server. WDYT?

Reply:
It's because the GenAI Quickstart is a zonal (Standard) cluster, so they'd need to modify the GenAI Quickstart cluster terraform. I can make instructions for setting up Agones within an existing cluster. Separately I can do a PR to get GenAI Quickstart onto an Autopilot cluster.